### PR TITLE
Soft 603 fwxv ci

### PR DESCRIPTION
**Pull request by ShiCheng-Lu**

### CODE REVIEW

**Comment by ryandancy** [comment]
Looks good, did you test to make sure it fails on both format and lint failures?

**Comment by ShiCheng-Lu** [comment]
I don't know why this is failing, llint c files doesn't seem to produce any output. python code could have raised an error somewhere.

File: libraries/ms-freertos/inc/FreeRTOSConfig.h [hunk 1]
```diff
@@ -61,7 +61,7 @@ extern uint32_t SystemCoreClock;
 #define INCLUDE_vTaskDelete 1
 #define INCLUDE_vTaskSuspend 1
 #define INCLUDE_xResumeFromISR 1
-#define INCLUDE_vTaskDelayUntil 1
+#define INCLUDE_xTaskDelayUntil 1
```
Comment (mitchellostler) [inline libraries/ms-freertos/inc/FreeRTOSConfig.h:64]: Do you know why this was changed? I'm pretty sure the API is **v**TaskDelayUntil

Comment (ShiCheng-Lu) [reply libraries/ms-freertos/inc/FreeRTOSConfig.h:64]: /* INCLUDE_vTaskDelayUntil was replaced by INCLUDE_xTaskDelayUntil.  Backward
 * compatibility is maintained if only one or the other is defined, but
 * there is a conflict if both are defined. */
 comment from FreeRTOS,
 the function itself is called xTaskDelayUntil which replaced an older vTaskDelayUntil. This Config was old so it still had vTaskDelayUntil but we should be using xTaskDelayUntil

**Comment by ShiCheng-Lu** [comment]
Oh right, since I've changed scons to use cpplint instead of the cpplint.py file renamed to lint.py, this linting via `scons lint` now requires cpplint to be installed via pip. `pip install --upgrade cpplint`
(or you can switch to ubuntu-20.04 box which has everything that's needed installed)

**Review by ryandancy — APPROVED** [review]
If we're using cpplint instead of lint.py, should we delete lint.py?

**Pull request merged**